log4j-core-test/src/test/java/org/apache/logging/log4j/core/ShutdownDisabledTest.java

```diff
@@ -17,16 +17,24 @@
 package org.apache.logging.log4j.core;
 
 import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertNull;
 
+import java.lang.reflect.Field;
 import org.apache.logging.log4j.core.config.Configuration;
 import org.apache.logging.log4j.core.test.junit.LoggerContextSource;
+import org.apache.logging.log4j.core.util.ReflectionUtil;
+import org.apache.logging.log4j.test.junit.SetTestProperty;
 import org.junit.jupiter.api.Test;
 
+@SetTestProperty(key = "log4j2.is.webapp", value = "false")
```

Comment:
```suggestion
@SetTestProperty(key = "log4j2.isWebapp", value = "false")
```

_Nit_: both work, but [`log4j2.isWebapp`](https://logging.apache.org/log4j/2.x/manual/systemproperties.html#log4j2.isWebapp) is the one we document on our website.